### DISCUSSION TITLE
Q About ex2

### DISCUSSION THREAD

**Discussion by AvinoamNukrai** [product-jam-2025/course#10] [General]
Hey, 
Could you clarify whether we’re expected to create a new project for the tic-tac-toe game or integrate it into the existing starter kit? 
![image](https://github.com/user-attachments/assets/46e7fae6-dea8-4e27-bf5b-baf0367aff42)

Thanks!

**Comment by jonshamir** [comment]
Hey, you should implement it inside the given starter kit
